server/integration/versions/pom.xml

```diff
@@ -23,6 +23,8 @@
       <maven.compiler.target>1.8</maven.compiler.target>
       <maven.compiler.source>1.8</maven.compiler.source>
       <server.output.dir.prefix>infinispan-server</server.output.dir.prefix>
+      <infinispan.module-suffix>-${project.artifactId}</infinispan.module-suffix>
+      <ansi.strip/>
```

Comment:
Given that this declaration is empty - do we need it at all?

Reply:
Yes, I think if the declaration was missing the fork would see `System.getProperty("ansi.strip") == "${ansi.strip}"`. Still, the versions POM now inherits these properties from the parent, so it shouldn't need to duplicate them.